crates/conformance-test-utils/src/madara.rs

```diff
@@ -0,0 +1,79 @@
+use eyre::Result;
+use kakarot_rpc_core::client::constants::STARKNET_NATIVE_TOKEN;
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_storage_var_address;
+
+use crate::types::{ContractAddress, StorageKey, StorageValue};
+
+/// Generates the genesis storage tuple for pre-funding a Starknet address on Madara.
+///
+/// This function calculates the storage key for the balance of the ERC20 Fee Token
+/// contract using the provided Starknet address. The resulting tuple represents the initial
+/// storage of the Fee Token contract, where the account associated with the Starknet address is
+/// pre-funded with the specified `amount`.
+pub fn genesis_fund_starknet_address(
+    starknet_address: FieldElement,
+    amount: FieldElement,
+    storage_offset_key: u64,
```

Comment:
what is this offset here? i guess that it's because the balance is stored as a Uint256, so two slots, but this function is indeed an hybrid between two functions:
- a first one could be a generic function to generate a proper tuple based on contract_address, storage_name, storage_offset
- a second one could use this first one to actually fund an account, and would hide the fact that the balance is a Uin256

Reply:
Actually when I was starting to do https://github.com/kkrt-labs/kakarot-rpc/issues/379, I thought the same to create this generic function that sets the storage and we use this to create abstraction for funding and setting kakarot storage 👍  I think it's a good idea to do this, let me update this